tests/network/vmi_multus.go

```diff
@@ -1041,7 +1042,19 @@ var _ = Describe("[Serial]SRIOV", func() {
 				migrationUID := tests.RunMigrationAndExpectCompletion(virtClient, migration, tests.MigrationWaitTime)
 				tests.ConfirmVMIPostMigration(virtClient, vmi, migrationUID)
 
-				Expect(checkMacAddress(vmi, interfaceName, mac)).To(Succeed(),
+				var interfaceName string
+
+				// It may take some time for the VMI interface status to be updated with the information reported by
+				// the guest-agent.
+				Eventually(func() error {
+					var err error
+					vmi, err = virtClient.VirtualMachineInstance(vmi.Namespace).Get(vmi.Name, &metav1.GetOptions{})
```

Comment:
I think you're updating the `vmi` variable as a side-effect here. Is that by design ? And is there a need for it ? AFAIU, you're not using it outside this scope.

Isn't the `vmi` variable explicitly updated elsewhere ? 

I'd prefer if "whatever happens in Eventually, stays in Eventually" .

Reply:
Fair enough. Done